study.md

```diff
@@ -246,5 +247,6 @@ Pick a few design heuristics. Describe some common, named refactorings you can
 use approach a good design.
 
 ```md
-<!-- your answer here -->
+ In Ruby, one can use inheritance and mixin to extend objects and follow the
+Open Closed Principle. In order to apply the Single Responsibility Principle, one can make objects that have one responsibility.
```

Comment:
Great start. What would you use to DRY up duplicated logic? Maybe "Extract Method". Have a look at some named refactorings, like "Extract Method", to level up.